### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by smolinari** [alexandrevilain/temporal-operator#745] [General]
Hi,

When will the next release be out? I'm waiting impatiently. :grin: 

Scott

**Comment by alexandrevilain** [comment]
Hi @smolinari ! 

We're cooking it up :) 

I'm currently waiting for CI to pass on:
- https://github.com/alexandrevilain/temporal-operator/pull/736
- https://github.com/alexandrevilain/temporal-operator/pull/744

Then I think that 0.19.0 will be out in the coming days

**Reply by smolinari** [reply]
That's great. I hoping sooner than later, because I've been waiting for this fix.  https://github.com/alexandrevilain/temporal-operator/pull/732

Scott

**Reply by alexandrevilain** [reply]
Release is now available: https://github.com/alexandrevilain/temporal-operator/releases/tag/v0.19.0

**Reply by smolinari** [reply]
Yippee! I'll report back, if all goes well (or not). :partying_face: 

Thanks for the work! :+1: 

Scott
